src/puppetlabs/pcp/message.clj

```diff
@@ -159,10 +163,21 @@
 (def descriptor-codec
   (b/compile-codec :byte encode-descriptor decode-descriptor))
 
+(def length-codec
+  (b/compile-codec :int-be
+                   (fn [l]
```

Comment:
Yeah, it can be replaced with the `(identity )` function. Originally I had type hints on the functions but I had to drop them as it turns out you can't do:
```clojure
(fn ^int [^int x] x)
```
in clojure:
```
CompilerException java.lang.IllegalArgumentException: Only long and double primitives are supported, ...
```